include/swift/SIL/SILLocation.h

```diff
@@ -755,17 +755,18 @@ class SILDebugScope;
 
 /// A SILLocation paired with a SILDebugScope.
 class SILDebugLocation {
-  const SILDebugScope *Scope = nullptr;
-  SILLocation Location;
+  const SILDebugScope *debugScope;
```

Comment:
The original code used the LLVM naming convention. Do we now have a Swift naming convention that's different from the LLVM one?

Reply:
Yes. Swift does camelCase for variables. We are slowly changing over time.

Reply:
And LLVM is changing more to that style as well